tools/build_release_uvision.bat

```diff
@@ -32,4 +32,4 @@ progen generate -t uvision -b
 SET LEVEL=%ERRORLEVEL%
 python tools/copy_release_files.py
 if %errorlevel% neq 0 exit %errorlevel%
-exit %level%
+exit /B %level%
```

Comment:
Nitpick - no newline at end of file
(don't feel like you need to fix this, just thought I would mention it)